evap/evaluation/migrations/0148_evaluation_main_language.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 5.1.3 on 2024-11-25 20:19
+
+from django.db import migrations, models
+
+
+def _migrate(apps, schema_editor):
+    Evaluation = apps.get_model("evaluation", "Evaluation")
+    for evaluation in Evaluation.objects.filter(state__gte=40):
+        evaluation.main_language = "de"
```

Comment:
Why? I would have expected "undecided" (which is the default value -> no explicit migration code needed?)

(If we keep this, can we use some named variable instead of the magic 40?)

Reply:
We want each published evaluation to have a decided main language. We check this during state transitions. So, we have to set a reasonable default for all evaluations which changing the main language is no longer possible.

Reply:
Ok, sounds reasonable. Could we still replace `40` here with some named constant?